contracts/zxp/ObjectRegistry.sol

```diff
@@ -4,7 +4,9 @@ pragma solidity ^0.8.19;
 import {IObjectRegistry} from "./interfaces/IObjectRegistry.sol";
 
 contract ObjectRegistry is IObjectRegistry {
-    bytes32 internal constant OWNER = "Owner";
+    bytes32 internal constant OWNER = "Owner"; // maybe put all constants in a library?

```

Comment:
Yeah that may be better. This way does show you exactly which ones are being used by a contract but that can still be accomplished.